scripts/report.sh

```diff
@@ -1,21 +0,0 @@
-#!/usr/bin/env bash
-
-set -x
-set -e
-set -o pipefail
-
-directory="/tmp"
-
-for file in "$directory"/*.tgz; do
```

Comment:
Hmm, it looks like this would work fine for a monorepo. Each package in the monorepo would create a separate tarball, which we're looping through here to generate the report

Reply:
Yeah, it appears that way too, but I just wanted to make sure before releasing this.